docs/guides/analyzers/feature_extraction.md

```diff
@@ -2,22 +2,41 @@
 hide:
   - footer
 ---
-The feature extraction analyzer creates attributes out of event data based on regular expressions. Different
-features can be specified in the `data/regex_features.yaml` file.
+The feature extraction analyzer creates attributes out of event data based on
+different extraction plugins.
 
-Please be aware that this analyzer does *not* extract ipv4, email-addresses and similar from *all* events, but only those that match the query_string.
+Currently supported are:
+* Regular expression based extractions
+  * [Regex Extraction Plugin](#regex-extraction-plugin)
+* Plaso parsed windows event logs
+  * [Winevt Extraction Plugin](#winevt-extraction-plugin)
+
+> **Note**
+Please be aware that this analyzer does *not* extract ipv4, email-addresses and
```

Comment:
s/email-addresses/email addresses

(and in other places in the doc) - https://learnenglish.britishcouncil.org/business-english/english-emails/unit-1-email-addresses